Project.toml

```diff
@@ -10,7 +10,6 @@ ExaTron = "28b18bf8-76f9-41ea-81fa-0f922810b349"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
```

Comment:
Do we depend on JLD2? This is large dep, and it would be nice to remove it if we can

Reply:
Removed.